cni-plugin/integration/flannel/flannel_test.go

```diff
@@ -0,0 +1,93 @@
+package flannel
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"testing"
+)
+
+const (
+	ConfigDirectory = "/var/lib/rancher/k3s/agent/etc/cni/net.d"
+	FlannelConflist = "10-flannel.conflist"
+)
+
+func ls(dir string, t *testing.T) []string {
```

Comment:
As an alternative, we could return an error instead of relying on `t`. This would maybe make it a bit clearer that the function is purely a utility, but it does mean we will end up handling more error cases in our tests (copy&pasta ftw).

Not bothered either way, will leave it up to you :D